content/en/docs/ibm/create-cluster.md

```diff
@@ -128,12 +189,150 @@ ibmcloud ks cluster create ${WORKER_NODE_PROVIDER} \
+    ```
+   If the above list contains the subnet corresponding to your VPC, that can be used to deploy your cluster - note it's
+   ID.
+   
+    b) To create a new subnet
+      * List address prefixes and note the CIDR block corresponding to a Zone,
+   in below example, for Zone: `us-south-3` CIDR block is : `10.240.128.0/18`
+         ```shell
+          $ ibmcloud is vpc-address-prefixes $VPC_ID
+          
+          Listing address prefixes of vpc r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx under account IBM as user new@user-email.com...
+          ID                                          Name                                CIDR block        Zone         Has subnets   Is default   Created   
+          r006-xxxxxxxx-4002-46d2-8a4f-f69e7ba3xxxx   rising-rectified-much-brew          10.240.0.0/18     us-south-1   false         true         2021-03-05T14:58:39+05:30   
+          r006-xxxxxxxx-dca9-4321-bb6c-960c4424xxxx   retrial-reversal-pelican-cavalier   10.240.64.0/18    us-south-2   false         true         2021-03-05T14:58:39+05:30   
+          r006-xxxxxxxx-7352-4a46-bfb1-fcbac6cbxxxx   subfloor-certainly-herbal-ajar      10.240.128.0/18   us-south-3   false         true         2021-03-05T14:58:39+05:30  
+         ```
+   * Now create a subnet as follows.
+   
+       ```shell
+       $ ibmcloud is subnet-create my-subnet $VPC_ID us-south-3 --ipv4-cidr-block "10.240.128.0/18"
+       Creating subnet my-subnet in resource group kubeflow under account IBM as user new@user-email.com...
+                              
+       ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
+       Name                my-subnet
+       ```
+
+   **Export the subnet id as,**
+   
+   `export SUBNET_ID=0737-27299d09-1d95-4a9d-a491-a6949axxxxxx`
+
+5. Create a `vpc-gen2` based Kubernetes cluster
+    
+    ```shell
+    $ ibmcloud ks cluster create ${WORKER_NODE_PROVIDER} \
+    --name=$CLUSTER_NAME \
+    --zone=$CLUSTER_ZONE \
+    --version=${KUBERNETES_VERSION} \
+    --flavor ${WORKER_NODE_FLAVOR} \
+    --vpc-id ${VPC_ID} \
+    --subnet-id ${SUBNET_ID} \
+    --workers=2
+    ```
+
+6. Attach a public gateway
+
+   This step is mandatory for Kubeflow deployment to succeed, because pods need public internet access to download images.
+   
+   First check, if your cluster is already assigned a public gateway:
+   
+   ```shell
+   $ ibmcloud is pubgws
+   Listing public gateways for generation 2 compute in all resource groups and region ...
+   ID                                          Name                                       Status      Floating IP      VPC                 Zone         Resource group   
+   r006-xxxxxxxx-5731-4ffe-bc51-1d9e5fxxxxxx   my-gateway                                 available   xxx.xxx.xxx.xxx       my-vpc              us-south-3   default   
+   
+   ```
```

Comment:
```suggestion
    ```shell
    ibmcloud is pubgws
    ```

    Example output:

    ```
    Listing public gateways for generation 2 compute in all resource groups and region ...
    ID                                          Name                                       Status      Floating IP      VPC                 Zone         Resource group   
    r006-xxxxxxxx-5731-4ffe-bc51-1d9e5fxxxxxx   my-gateway                                 available   xxx.xxx.xxx.xxx       my-vpc              us-south-3   default   
   
    ```
```